cmd/client/main.go

```diff
@@ -239,17 +243,17 @@ var (
 						Usage:    "Duration of timeout, in seconds",
 						Required: true,
 					},
-					daemonAddrFlag,
+					swapdPortFlag,
 				},
 			},
 		},
-		Flags: []cli.Flag{daemonAddrFlag},
```

Comment:
I wanted users to be able to specify `--swapd-port` before or after the subcommand.  The `spf13/cobra` library makes this easy to do, but I guess it is not supported by urfave: https://github.com/urfave/cli/issues/1391

When we define the same flag globally and after the command, the behavior is bad. I wasn't getting an error for the global flag (since it was listed as a global flag), but when reading it, we would get the defaulted value from the flag to the subcommand instead of the globally set value.
Example:
```
./swapcli --swapd-port 5432 address
```
The actual port used would be 5001, because the defaulted value for `--swapd-port` after the `address` subcommand was taking precedence over the 5432 value.